### PR TITLE
Issue #3. Update implementation of hook_block_view_MODULE_DELTA_alter

### LINKED ISSUE #3: Unsupported operand type error report.
I get the error message
`Error: Unsupported operand types in custom_header_block_header_block() (line 28 of ...../modules/custom_header_block/custom_header_block.module).`

### PR DESCRIPTION
**Pull request by argiepiano**
Fixes #3

### CODE REVIEW

File: custom_header_block.module [hunk 1]
```diff
@@ -34,11 +35,13 @@ function custom_header_block_header_block($settings, $header_style) {
   $site_config = config('system.core');
   $style = config_get('custom_header_block.settings', 'style');
   
-  $variables['slogan_bottom'] = ($style == 'slogan_bottom_flipped' || $style == 'slogan_bottom')?TRUE:FALSE;
-  $variables['logo']        = $settings['logo'] ? backdrop_get_logo() : NULL;
-  $variables['site_name']   = $settings['site_name'] ? check_plain($site_config->get('site_name')) : '';
-  $variables['site_slogan'] = $settings['site_slogan'] ? filter_xss_admin($site_config->get('site_slogan')) : '';
-  $variables['front_page']  = url();
+  $variables['slogan_bottom']   = ($style == 'slogan_bottom_flipped' || $style == 'slogan_bottom')?TRUE:FALSE;
```
Comment (klonos) [inline custom_header_block.module]: Can we please also fix this while at it?:
```suggestion
  $variables['slogan_bottom']   = ($style == 'slogan_bottom_flipped' || $style == 'slogan_bottom') ? TRUE : FALSE;
```

Comment (argiepiano) [reply custom_header_block.module]: Fixed

**Comment by docwilmot** [comment]
Thanks folks, but sadly I cant merge, no access. 😞 
@klonos can you please grant me access and/or merge this?

File: custom_header_block.module [hunk 2]
```diff
@@ -34,11 +35,12 @@ function custom_header_block_header_block($settings, $header_style) {
   $site_config = config('system.core');
   $style = config_get('custom_header_block.settings', 'style');
   
-  $variables['slogan_bottom'] = ($style == 'slogan_bottom_flipped' || $style == 'slogan_bottom')?TRUE:FALSE;
-  $variables['logo']        = $settings['logo'] ? backdrop_get_logo() : NULL;
-  $variables['site_name']   = $settings['site_name'] ? check_plain($site_config->get('site_name')) : '';
-  $variables['site_slogan'] = $settings['site_slogan'] ? filter_xss_admin($site_config->get('site_slogan')) : '';
-  $variables['front_page']  = url();
+  $variables['slogan_bottom']   = ($style == 'slogan_bottom_flipped' || $style == 'slogan_bottom')? TRUE : FALSE;
```
Comment (VasasA) [inline custom_header_block.module]: A space is missing before the '?' mark.
Thank You for the PR.

Comment (argiepiano) [reply custom_header_block.module]: Fixed

**Pull request merged**